ROADMAP.md

```diff
@@ -0,0 +1,85 @@
+# Rustls development roadmap
+
+## Future priorities
+
+In rough order of priority:
+
+* **FIPS Certification for Default Cryptographic Library**.
+  Change the default cryptographic library to something with FIPS certification.
+  rustls/rustls#1540
+
+* **Comprehensive Performance Benchmarking**.
+  Performance should be a headline feature of Rustls. We need to develop a more
+  comprehensive benchmarking system so that we can assess and improve performance
+  from multiple angles, including CPU usage, latency, and memory usage.
+
+* **Add No-Allocation / Write-Through API**.
+  Would make handshakes faster and give the caller more control over allocations.
+  RFC: rustls/rustls#1420
+
+* **Support no_std**.
+  Enables use of rustls in more memory-constrained environments.
+  RFC: rustls/rustls#1399
+
+* **OpenSSL API Compatibility Layer**.
+  Add an OpenSSL C API compatibility layer for adoption purposes.
+
+* **Support Encrypted Client Hello**.
+  Encrypted Client Hello is an upcoming standard from the TLS WG providing better
+  production for some of the data sent by a client in the initial ClientHello
```

Comment:
one typo: "production" -> "protection"
one editorial change: we refer to "Client Hello" earlier, I think we should keep using that consistently (or put the `ClientHello` instance in backticks).
```suggestion
  protection for some of the data sent by a client in the initial Client Hello
```